components/tooltip.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {CSSProperties} from 'react';
+import {Tooltip as RBTooltip} from 'react-bootstrap';
+
+type Props = {
+    id?: string;
+    className?: string;
+    style?: CSSProperties;
+    children?: React.ReactNode;
+    positionLeft?: number;
+    placement?: string;
+};
+
+export default function Tooltip(props: Props) {
```

Comment:
We could consider wrapping this in a `React.forwardRef` so that you can still access the underlying tooltip, but that's not really necessary unless we actually use that ref anywhere. 0/5 on changing anything now, but we might consider that if we start to do this in more places

Reply:
I tried to make it as minimal as possible, only exposing what is needed now.

Reply:
Oh yeah, that makes sense. I guess this isn't like where we'd be forwarding a ref to a DOM element.